<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellTypeDialog</class>
 <widget class="QDialog" name="CellTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="a0_btn_">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a1_btn_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a2_btn_">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a3_btn_">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a4_btn_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a5_btn_">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a6_btn_">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a7_btn_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a8_btn_">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closed_btn_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bomb_btn_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Closed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark &quot;Bomb&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Out of field</string>
   </property>
  </widget>
  <widget class="QLabel" name="cell_img_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bomb (Death)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="death_btn_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrong_mine_btn_">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wrong Mine</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closed_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bomb_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>out_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a0_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a1_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a2_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a4_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a3_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a5_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a6_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a7_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a8_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>death_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wrong_mine_btn_</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDialog</receiver>
   <slot>OnSelectType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSelectType()</slot>
 </slots>
</ui>
